src/components/transactions/message-view/StakeTxDetails.vue

```diff
@@ -1,45 +1,36 @@
 <template>
   <div class="tx__content">
-    <TransactionIcon
-      :transaction-group="transaction.group"
-      :transaction-type="type"
-    />
+    <TransactionIcon :transaction-type="type" />
     <div class="tx__content__left">
       <h3>{{ caption }}</h3>
       <span>With&nbsp;</span>
-      <router-link
-        :to="`/staking/validators/${transaction.value.validator_address}`"
-      >
+      <router-link :to="`/staking/validators/${transaction.details.to[0]}`">
```

Comment:
this will conflict with the new url schema

Reply:
Oki, you mean that can be replaced by:

```javascript
<router-link :to="`/validators/${transaction.details.to[0]}`">
```